### PR TITLE
Added tests for CLI

### PR DESCRIPTION
**Pull request by esikachev**
Added tests for testing workflow

### CODE REVIEW

**Comment by esikachev** [comment]
Recheck this patch after merge esikachev/my-dev-server#62 and esikachev/my-dev-ci-config#21

File: my_dev/base.py [hunk 1]
```diff
@@ -22,7 +22,8 @@ def ssh(self):
         user_id = user.get_id(CONF.username)
         client = ssh.Ssh(user_id)
         if client.get(host).get('status_code') == 404:
-            password = getpass('Enter the password for ssh connection: ')
+            password = self.enter_pass('Enter the password for ssh '
```
Comment (zemuvier) [inline my_dev/base.py]: Why did you change it?

File: my_dev/config.py [hunk 2]
```diff
@@ -7,17 +7,21 @@
 
 opts = [
     cfg.StrOpt('host',
-               default='http://localhost:5000',
+               default='http://127.0.0.1:5000',
```
Comment (zemuvier) [inline my_dev/config.py]: So why did you change this?
